client/blocks/comment-detail/docs/comment-faker.jsx

```diff
@@ -89,6 +89,41 @@ export const CommentFaker = WrappedCommentList => class extends Component {
 		} } );
 	}
 
+	submitComment = comment => {
+		const { comments } = this.state;
+		const newCommentId = parseInt( maxBy( keys( this.state.comments ) ) + 1, 10 );
```

Comment:
also why not use something like `lodash.uniqueId`? don't want much logic here in the faker